extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -177,24 +187,21 @@ public function captcha() {
 		$this->rawHtml = '';
 		$this->isFancyCaptcha = false;
 
-		if ( $this->shouldDisplayCaptcha() ) {
-			$captchaObj = $this->getCaptchaObj();
-			if ( !empty( $captchaObj ) ) {
-				$this->rawHtml = $captchaObj->getForm();
-				$this->isFancyCaptcha = (
-					class_exists( 'Captcha\Module\FancyCaptcha' ) &&
```

Comment:
We talked about this previously:

https://github.com/Wikia/app/pull/6449/files#diff-fd432f58d5c8919e5fe6b60bacf809ecR185